packages/@aws-cdk/aws-servicecatalog/lib/product-stack.ts

```diff
@@ -1,11 +1,25 @@
 import * as crypto from 'crypto';
 import * as fs from 'fs';
 import * as path from 'path';
+import { IBucket } from '@aws-cdk/aws-s3';
 import * as cdk from '@aws-cdk/core';
 import { Construct } from 'constructs';
 import { ProductStackSynthesizer } from './private/product-stack-synthesizer';
+import { ProductStackAssetBucket } from './product-stack-asset-bucket';
 import { ProductStackHistory } from './product-stack-history';
 
+/**
+ * Product stack props.
+ */
+export interface ProductStackProps {
+  /**
+   * Product stack asset bucket.
+   *
+   * @default - None
```

Comment:
What does default - none mean? No asset bucket created?

Reply:
Yes I updated it to be slightly more descriptive, but the asset bucket is only generated if it is needed, otherwise there is no asset bucket.